src/sagas/import-project.saga.js

```diff
@@ -71,6 +72,11 @@ export function* handleImportError(err: Error): Saga<void> {
 }
 
 export function* importProject({ path }: Action): Saga<void> {
+  const store = window.store.getState();
```

Comment:
Ah, so `window.store` is meant to be used for debugging only. When we're within a saga, there are more idiomatic ways to get the state.

In this case, it'd be `yield select(selectorFunction)`.

You can use the selector function I mentioned in an earlier function, to do:

```js
const isOnboardingCompleted = yield select(getOnboardingCompleted);
```

Reply:
🤦‍♀️ Much nicer.